config/schemas/clientSchema.js

```diff
@@ -0,0 +1,20 @@
+const z = require("zod");
+require("dotenv").config();
+
+const envSchema = z.object({
+  NEXT_PUBLIC_BASE_URL: z.string().url().default("http://localhost:3000"),
+});
+
+const clientEnv = envSchema.safeParse(process.env);
+
+if (!clientEnv.success) {
+  console.error("There is an error with the client environment variables");
+  console.error(clientEnv.error.issues);
```

Comment:
Could we use the `logger` here instead of console?

Reply:
It gives the `SyntaxError: Cannot use import statement outside a module` if I do it. I can't use `import/export` in the files I create, even though other files use it. It's weird.